src/schema/schema_create.c

```diff
@@ -1327,6 +1328,17 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
     __wt_free(session, import_list.entries);
     __wt_free(session, export_file);
 
+import_err:
+    /*
+     * Handle the logic to increment the create import statistics separately here as the
+     * clear_import_flag variable will not be set if the config checking fails.
+     */
+    if (import) {
```

Comment:
`import` will be `true` in recursive calls too. This makes statistics invalid.

Reply:
Thanks Sue, this is much simpler and makes a lot more sense too.